lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -142,6 +143,8 @@ type Client struct {
 	// we reuse the same image to avoid allocating on each bitmap
 	img *image.RGBA
 
+	bitmapMsgBuffer []byte
```

Comment:
This is being used exclusively as the buffer for `PNG2Frame`s, so I would call it something like `PNG2FrameBuffer` and add a comment explaining why it's here.

Reply:
Yeah png2FrameBuffer is a better name for this buffer. I've also added a comment.